src/packages/client/helpers/build.js

```diff
@@ -31,8 +32,15 @@ async function main() {
       target: 'node12',
       outfile: 'generator-build/index.js',
       entryPoints: ['src/generator.ts'],
-      external: ['_http_common']
     }),
+    // copy wasm files, etc necessary for undici
+    fs.copy(
```

Comment:
I'm not familiar enough with esbuild. The problem is that undici uses `path.resolve` to establish a path to wasm files (https://github.com/nodejs/undici/blob/e705509ab22ab80aadf0fc1a394afff7dc014fdf/lib/client.js#L390-L398) and I wasn't able to find a plugin that could resolve that and copy files accordingly.